templates/applications/application.html

```diff
@@ -4,136 +4,198 @@
-                    {% if "stage_progress" in application and application["stage_progress"]["application"] %}<i class="p-icon--success"></i> 
-                    {% else %}<i class="p-icon--information"></i> 
-                    {% endif %}
-                    Application review
-                </h4>
-                <p>Initial resume screening. A fully trained hiring lead will review your application against the job criteria set for this role.</p>
-            </li>
-            <li class="p-list_item">
-                <h4 class='u-no-margin--bottom'>
-                    {% if "stage_progress" in application and application["stage_progress"]["assessment"] %}<i class="p-icon--success"></i> 
-                    {% else %}<i class="p-icon--information"></i> 
-                    {% endif %}
-                    Assessment
-                </h4>
-                <div>
-                    There are three parts to this stage:
-                    <ol>
-                        <li>
-                            <p class='u-no-margin--bottom'>A written interview, submitted and reviewed anonymously to reduce bias. This is your opportunity to highlight your key strengths for this position and once reviewed will be referred back at later stage interviews.</p>
-                        </li>
-                        <li>
-                            <p class='u-no-margin--bottom'>A psychometric assessment measuring fluid intelligence and behavioural traits.</p>
-                        </li>
-                        <li>
-                            <p class='u-no-margin--bottom'>Initial Meet & Greet Interview with a member of Canonical to share more about life at Canonical and to answer some of your initial basic questions.</p>
-                        </li>
-                    </ol>
-                </div>
-            </li>
-            <li class="p-list_item">
-                <h4 class='u-no-margin--bottom'>
-                    {% if "stage_progress" in application and application["stage_progress"]["early_stage"] %}
-                    <i class="p-icon--success"></i> 
-                    {% else %}<i class="p-icon--information"></i> 
+          <li class="p-list_item">
+            <h4 class='u-no-margin--bottom'>
+              {% if "stage_progress" in application and application["stage_progress"]["application"] %}<i class="p-icon--success"></i> 
+              {% else %}<i class="p-icon--information"></i> 
+              {% endif %}
+              Application review
+            </h4>
+            <p>Initial resume screening. A fully trained hiring lead will review your application against the job criteria set for this role.</p>
+          </li>
+          <li class="p-list_item">
+            <h4 class='u-no-margin--bottom'>
+              {% if "stage_progress" in application and application["stage_progress"]["assessment"] %}<i class="p-icon--success"></i> 
+              {% else %}<i class="p-icon--information"></i> 
```

Comment:
Can you please add a11y text within the icon. This will be hidden from the screen but will be read by a screen reader.